rs/cli/src/general.rs

```diff
@@ -51,12 +73,41 @@ pub(crate) async fn vote_on_proposals(
         print!("\x1B[1A\x1B[K");
         std::io::stdout().flush().unwrap();
         for proposal in proposals_to_vote.into_iter() {
+            let content = proposal.proposal.as_ref().unwrap().action.as_ref().unwrap();
+            let parsed = match content {
+                ic_nns_governance::pb::v1::proposal::Action::ExecuteNnsFunction(ExecuteNnsFunction {
+                    nns_function: _,
+                    payload,
+                }) => match Decode!(&payload, UpdateSubnetReplicaVersionPayload) {
+                    Ok(payload) => payload,
+                    Err(e) => {
+                        error!("Couldn't decode into update subnet replica version payload: {:?}", e);
+                        continue;
+                    }
+                },
+                _ => {
+                    warn!("Proposal's content wasn't correct. Skipping...");
+                    continue;
+                }
+            };
+
+            if parsed.replica_version_id != *latest_version
+                && !Confirm::new()
+                    .with_prompt(format!("CAUTION! You are about to vote on a proposal that will update the subnet '{}' to a non-latest version! Do you want to continue?", parsed.subnet_id.0.to_string().split('-').collect::<Vec<_>>().first().unwrap()))
```

Comment:
This could be a bit annoying if we're rolling out two versions at the same time.
But I suppose "remembering" such version for the duration of the single run could be a good compromise.
WDYT?